src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+public class ColorSupplier {private Random random = new Random();
```

Comment:
Code quality issue: According to the Java Code Convention, each class member should be declared on a new line.